test/rust_src/src/indent-tests.el

```diff
@@ -4,6 +4,31 @@
 
 (require 'ert)
 
+(ert-deftest move-to-column ()
```

Comment:
Great tests. I think it would be better if they were broken up into smaller functions so it was easier to debug a failure.

Reply:
Done. I got by by using the backtrace to see where things went wrong, but I can see how breaking it up makes it a lot easier to quickly see what misbehaved.